test/ssltest_old.c

```diff
@@ -1891,7 +1891,7 @@ int doit_localhost(SSL *s_ssl, SSL *c_ssl, int family, long count,
          */
 
         /*
-         * We have non-blocking behaviour throughout this test program, but
+         * We have non-blocking behavior throughout this test program, but
```

Comment:
I think comments can be in whatever variant of English the original author wants :)  We don't have a style guidelines for comment text.  I thought "initialise" was valid English, but you corrected me.  Shrug, whatever.  We've already spent too much time on this. :)